tests/gh-actions/install_multi_tenancy.sh

```diff
@@ -7,3 +7,6 @@ kubectl -n kubeflow wait --for=condition=Ready pods -l kustomize.component=profi
 
 echo "Installing Multitenancy Kubeflow Roles"
 kustomize build common/kubeflow-roles/base | kubectl apply -f -
+
+echo "Installing Multitenancy Network policies"
+kustomize build common/networkpolicies/base | kubectl apply -f -
```

Comment:
@biswajit-9776 Lets salvage what  we have in this PR except for the training-operator changes and add below the networkpolicies the static and dynamic PSS stuff in your own PR.

I will probably close this PR here afterwards and reopen another one for the training operator test restructuring